models/tickets.py

```diff
@@ -52,8 +49,11 @@ def json(self):
         assignedUserData = UserModel.find_by_id(self.assignedUser)
         assignedUser = "{} {}".format(assignedUserData.firstName, assignedUserData.lastName)
 
-        dateTimeNow = datetime.now()
-        minsPastUpdate = int((datetime.now() - self.updated).total_seconds() / 60)
+        if self.updated_at is None:
+            minsPastUpdate = 0
```

Comment:
This branch will never trigger. Now that `updated_at` is defaulted to the time of initialization.